README.md

```diff
@@ -128,5 +142,5 @@ Tool | Android | Java | Documentation |
 
 #### Support for sharable configurations
 
-We plan to add support for consuming rules (eg: configuration files for Checkstyle or PMD, default exclude filters, etc) via a
+The plugin can consume rules (eg: configuration files for Checkstyle or PMD, default exclude filters, etc) via a
 separate artifact you can share across projects. _More info to come._
```

Comment:
Not sure. I'd like to expand this with an example, but we have no publicly deployed artifact for shared rules yet, hence the `More info to come` still there.